static/js-src/feature-page.js

```diff
@@ -0,0 +1,71 @@
+// Event handler. Used in feature.html template.
+const subscribeToFeature = (featureId) => { // eslint-disable-line no-unused-vars
```

Comment:
do you need this eslint-disable comment?

Reply:
Yes, subscribeToFeature is used in the feature.html Jinja template. ESLint doesn't read that, so it throws "subscribeToFeature" is declared but not used. Same below.

Reply:
what if you exported it?

Reply:
It's directly used in the django template like "onclick="subscribeToFeature(...)", and all files in /static/js-src are inlined by django to each html files.
If I understand correctly, we need to change them to modules, and add workflow to generate nomodule to the template. Seems not worth it for this lint exception.

Reply:
I'm just wondering if exporting the variable silenced the warning and indicated that it's used elsewhere at the same time?

Reply:
Oh, I see. Seem working. Updated. Thanks!